### PR TITLE
Nanostring visualization

### LINKED ISSUE #25: New visualization for nanostring data
This issue is mainly a placeholder to sort out details, but we'd like to add an additional visualization tab similar to the correlation analysis shown here: https://nanostring-model-ad.jax.org/ (but we probably want to simplify for our audience)

We have code for that visualization in an email from Ravi Pandey, who built it. Potentially we could reuse some of it, with appropriate credit.

### PR DESCRIPTION
**Pull request by sharlagelfand**
Finally have the nanostring visualization in a good place 🎉  Closes #25.

I'm pretty happy with how this is looking, except the plot may be a bit small - had to adjust some of the theme sizing to have it fit (especially the cluster label for cluster A, on the left, since the width is narrower). Let me know if you think it's too small and I'm happy to think of a solution (maybe have the plot overflowing the page div, which is fixed at 1000px across the different pages).

Hopefully all the modeling and data cleaning is correct - please let me know if anything looks off! Might be helpful to have someone look it over.

Thanks!

### CODE REVIEW

File: R/magora_corrplot.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#'
+#' @return A ggplot2 object.
+magora_corrplot <- function(data) {
+
+  ggplot2::ggplot() +
+    ggplot2::geom_tile(data = data, ggplot2::aes(x = .data$module, y = .data$model_sex), colour = "black", fill = "white") +
+    ggplot2::geom_point(data = dplyr::filter(data, .data$significant), ggplot2::aes(x = .data$module, y = .data$model_sex, colour = .data$estimate, size = abs(.data$estimate))) +
+    ggplot2::scale_x_discrete(position = "top") +
+    ggplot2::scale_size(guide = "none", limits = c(0, 0.5)) +
+    ggplot2::scale_color_gradient2(limits = c(-0.5, 0.5), breaks = c(-0.5, 0, 0.5), low = "#2166AC", high = "#B2182B", name = "Correlation", guide = ggplot2::guide_colorbar(ticks = FALSE)) +
```
Comment (karawoo) [inline R/magora_corrplot.R:15]: Do you think we could use some of the sage colors here? Maybe "blueberry" (`#164B6E`) and "apple" (`#85070C`)?

**Review by karawoo — CHANGES_REQUESTED** [review]
🙌  this looks great! I have one small change below, and I will ask Ravi about possibly looking over the analysis portion.

File: R/magora_corrplot.R (hunk 1 above)
Comment (sharlagelfand) [reply R/magora_corrplot.R]: Ooh yes, done! Looks much better, thanks!

**Review by karawoo — APPROVED** [review]
Looks good to me, and we got the 👍  from Ravi via email
